distributor-api-v1/operations.md

```diff
@@ -123,48 +122,149 @@ Preferred initial call used to obtain all static data about distributor configur
+| `RequiredFields` | array of [RequiredField](operations.md#requiredfield) | required | Form fields which are required and need to be filled in. |
+| `ServiceId` | string | required | Unique identifier of the service to which the configuration is bound to. |
+
+#### PaymentCardInput
+
+* `NotRequested` - Payment card info is not requested.
+* `Requested` - Payment card info is requested, but not validated.
+* `Required` - Payment card info is requested and validated. 
+
+#### RequiredField
+
+* `Telephone`
+
+#### Enterprise 
+
+|  | Property | Type | Description |
+| :--- | :--- | :--- | :--- |
+| `Id` | string | required | Unique identifier of the enterprise. |
+| `AcceptedCurrencyCodes` | array of string | required | List of currency codes accepted by the enterprise. |
+| `AdditionalLegalStatements` | array of [LocalizedText](operations.md#localizedtext) | required | Additional legal statements. |
+| `Address` | [Address](operation.md#address) | required | Address of the enterprise. |
+| `Categories` | array of [RoomCategory](operations.md#roomcategory) | required | List of active room categories of the enterprise. |
+| `CityId` | string | required | Unique identifier of the city. |
+| `DefaultCurrencyCode` | string | required | Default enterprise currency code in ISO format. |
+| `DefaultLanguageCode` | string | required | Default enterprise language in ISO format. |
+| `DefaultRateCurrencyCode` | string | required | Default enterprise rate currency code in ISO format. |
+| `Description` | [LocalizedText](operations.md#localizedtext) | required | Enterprise description. |
+| `Email` | string | required | Email of the enterprise. |
+| `IanaTimeZoneIdentifier` | string | required | IANA time zone identifer. |
+| `ImageId` | string | optional | Unique identifier of the enterprise logo. |
+| `IntroImageId` | string | optional | Unique identifier of the enterprise intro image. |
+| `Name` | [LocalizedText](operations.md#localizedtext) | required | Enterprise name. |
+| `Pricing` | string [Pricing](operations.md#pricing) | required | Pricing method used by the enterprise. |
+| `PrivacyPolicyUrl` | [LocalizedText](operations.md#localizedtext) | required | Enterprise privacy policy URL. |
+| `Products` | array of [Product](operations.md#product) | required | List of active products which can be offered to the customer. |
+| `TaxEnvironmentCode` | string | required | Tax environment code. |
+| `Telephone` | string | required | Telephone of the enterprise. |
+| `TermsAndConditionsUrl` | [LocalizedText](operations.md#localizedtext) | required | Enterprise terms and conditions URL. |
+
+#### Address
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `City` | string | optional | City. |
+| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](operations.md#country). |
+| `Latitude` | number | optional | The latitude. |
```

Comment:
Why the latitude is so far from longitude?

Reply:
alphabetical sort, changed